### ISSUE TITLE
Login/Reset Password - error messages not following WET 

### ISSUE THREAD

**Issue by mmiron-HC** [mmiron-HC/WST-RSAMS#28]
URL: https://hcdev.openplus.ca/en/user/login
User Agent: chrome, Firefox
Language: Both

**Issue Description:** 

**Steps to reproduce:**
scenario 1: 
Click on "Login" button without completing any information.
generic inline error is displayed for only the username field.

scenario 2: 
incorrect username/password combination
click on "Login" button
non-descript error displayed above, Error message appears below the input field. 
![image](https://user-images.githubusercontent.com/65728700/128084483-3dd7a3c5-e513-4b23-8bcf-2cbb72d5111d.png)

scenario 3: 
Reset password page.
leave username or email address input field blank and trigger the "Submit" button
![image](https://user-images.githubusercontent.com/65728700/128084657-5d75be87-0735-4aa6-bc67-2f812acdfd23.png)

Expected results: In keeping with the WET common look and feel, error messages should be displayed at the top of the page with jump links to the proper error. 
Error messages should be part of the label. 
Error messages should be specific.
see: https://wet-boew.github.io/wet-boew/demos/formvalid/formvalid-en.html
![image](https://user-images.githubusercontent.com/65728700/128084293-bee5cc7c-e48f-40d0-a0ea-4dc1b8c2e9c5.png)

Currently, the Error message (top of page) is not being read by screen reader b/c the focus is on the input field that fails. If you apply the WET behaviour for error messages this should solve the issue. The focus will be at the top on error messages alert.

**Comment by davidgotlieb** [comment]
As per #105

We are using the default WxT error handling. From OP:

WxT is not 100% WET compliant. Currently, error message handling in WxT is not done the same as it WET. It hasn't been implemented yet. It will be a major undertaking to get the WET error message handling into WxT, when it does get there, it will be pulled into RSAMS as part of the regular WXT update.
